components/src/core/components/Form/form.scss

```diff
@@ -15,6 +15,14 @@
   &.--empty {
     min-height: $oxd-form-min-height;
   }
+  &.oxd-schema-form-container {
+    :deep(.oxd-grid-1),
+    :deep(.oxd-grid-2),
+    :deep(.oxd-grid-3),
+    :deep(.oxd-grid-4) {
```

Comment:
Do we only have grid classes upto 4?

Reply:
Yes, we have up to 4
https://github.com/gayan-orangehrm/oxd/blob/1e76d67e768492d3ee08688b13705044b5e5544d/components/src/core/components/Grid/grid.scss#L4